loopdb/loopout.go

```diff
@@ -34,9 +34,9 @@ type LoopOutContract struct {
 	// client sweep tx.
 	SweepConfTarget int32
 
-	// TargetChannel is the channel to loop out. If zero, any channel may
-	// be used.
-	UnchargeChannel *uint64
+	// OutgoingChanSet is the set of channels that may be used. If empty,
+	// any channel may be used.
+	OutgoingChanSet []uint64
```

Comment:
Consider changing the doc such that it says the items are short channel ids (like above).

Reply:
added short ids